airbyte-db/src/main/java/io/airbyte/db/jdbc/JdbcUtils.java

```diff
@@ -151,6 +151,19 @@ private static void setJsonField(ResultSet r, int i, ObjectNode o) throws SQLExc
     }
   }
 
+  /**
+   * In some sources Integer might have value larger than {@link Integer#MAX_VALUE}. E.q. MySQL has
+   * unsigned Integer type, which can contain value 3428724653. If we fail to cast Integer value, we
+   * will try to cast Long.
+   */
+  private static void putInteger(ObjectNode node, String columnName, ResultSet r, int i) {
```

Comment:
@DoNotPanicUA please use more significant naming for r and i. Another point should the method really be static?

Reply:
@yaroslav-hrytsaienko 
The names are fixes. 
About static - yes. it's utility class where all methods are static. As result, this method is been called only inside other static method.